libs/blocks/table/table.js

```diff
@@ -316,16 +296,16 @@ function handleScrollEffect(table) {
   } else {
     headingRow.classList.add('top-border-transparent');
   }
-  headingRow.style.top = `${gnavHeight + (highlightRow ? highlightRow.offsetHeight : 0)}px`;
+  const topOffset = gnavHeight + (highlightRow ? highlightRow.offsetHeight : 0);
```

Comment:
nit: I think this could be shorter
```suggestion
  const topOffset = gnavHeight + (highlightRow?.offsetHeight || 0);
```